src/service/ratelimit.go

```diff
@@ -286,6 +286,8 @@ func (this *service) ShouldRateLimit(
 		}
 
 		logger.Debugf("caught error during call")
+
+		logger.Error(err)
```

Comment:
This can log spam pretty badly. If we want to log something here I would have some kind of back off logger that warns periodically. Feel free to print the err as part of the debug log above.

Reply:
For now, I just added the error to the debug.